newsplease/crawler/simple_crawler.py

```diff
@@ -1,7 +1,74 @@
+# Based on https://github.com/adbar/trafilatura/blob/master/trafilatura/utils.py
+import logging
+import socket
 import copy
 import threading
 
-from six.moves import urllib
+try:
+    # this module is faster
+    import cchardet
+except ImportError:
+    cchardet = None
+# https://charset-normalizer.readthedocs.io/en/latest/
```

Comment:
why are these two links here? could you add a comment, stating why you put them, e.g., "more information on cchardet can be found at ..." also its not clear why ftfy is put here, since it isnt even imported

Reply:
Fixed and made cchardet essential. NOTE: I don't know how to test this part as I don't know which website is not using UTF-8 encoding.